TixFactory.Queueing.Remote/TixFactory.Queueing.Remote/Implementation/RemoteItemQueueFactoryV3.cs

```diff
@@ -58,6 +61,8 @@ public IRemoteItemQueue<TItem> CreateRemoteItemQueue<TItem>(string queueName, Ti
 					state: null,
 					dueTime: countRefreshInterval.Value,
 					period: countRefreshInterval.Value);
+
+				_RefreshCountTimers.Add(countRefreshTimer);
```

Comment:
I believe the `Timer` was getting garbage collected, so by adding it to the initialized class field it should keep it in memory.